.clang-format

```diff
@@ -1,16 +1,26 @@
 ---
-BasedOnStyle: Google
-Language: Cpp
-Standard: Cpp03
+# Indentation for all files
+ColumnLimit: 120
+IndentWidth: 4
 TabWidth: 4
 UseTab: Never
-ColumnLimit: 120
-NamespaceIndentation: All
+
+# C++ Options
+Language: Cpp
+Standard: c++17
+
+BasedOnStyle: Google
+
+IncludeBlocks: Preserve
 
 AccessModifierOffset: -4
 ContinuationIndentWidth: 4
-IndentWidth: 4
 
+# Constructor initializers better formatted in presence of preprocessor conditions (see image.cpp)
+BreakConstructorInitializers: AfterColon
+AllowAllConstructorInitializersOnNextLine: true
+
+# break braces for most of the cases (except conditionals and loops)
 BreakBeforeBraces: Custom
```

Comment:
IMHO this setup of `BreackBeforeBraces` adds a lot of extra vertical empty space that's not really useful. Can we stick to the default?

Reply:
You are right, this is where subjectivity starts. I have a slight preference for breaking these braces in most of cases, but I could adapt to the Google style over time 😇 .

I'll take another iteration with your proposed changes.